src/ViewModel/TextField.php

```diff
@@ -83,7 +83,7 @@ public static function emailInput(
         string $state = null,
         MessageGroup $messageGroup = null
     ) {
-        Assertion::eq(null, $formFieldInfoLink['alignLeft']);
+        Assertion::eq(null, $formFieldInfoLink['alignLeft'] ?? null);
```

Comment:
I support the upgrade of the libraries. I will just review when this property is not present and confirm that is expected.

Reply:
Yup, that'd be great. This code effectively reproduces the bahviour that was happening in previous versions of PHP without the warning (array read of non-set key = null), but the code didn't seem to be written expecting it. Thanks for looking into it.